src/api/versions/v1/services/websocket-service.ts

```
@@ -119,28 +119,32 @@ export class WebSocketService {
+
+    const typeId = binaryReader.unsignedInt8();
 
     switch (typeId) {
       case WebSocketType.PlayerIdentity: {
-        return this.handlePlayerIdentityMessage(user, payload);
+        return this.handlePlayerIdentityMessage(user, binaryReader);
       }
 
       case WebSocketType.Tunnel: {
-        return this.handleTunnelMessage(user, payload);
+        return this.handleTunnelMessage(user, binaryReader);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)